src/Migration/CitizenProjectCategorySkillMigration.php

```diff
@@ -0,0 +1,129 @@
+
+class CitizenProjectCategorySkillMigration
+{
+    private $entityManager;
+
+    const CATEGORIES_SKILLS_MAPPING = [
+        'C001' => [
+            ['skill' => 'S001', 'promotion' => true],
+            ['skill' => 'S002', 'promotion' => true],
+            ['skill' => 'S003', 'promotion' => true],
+            ['skill' => 'S004', 'promotion' => false],
+            ['skill' => 'S005', 'promotion' => false],
+            ['skill' => 'S006', 'promotion' => false],
+        ],
+        'C002' => [
+            ['skill' => 'S007', 'promotion' => true],
+            ['skill' => 'S008', 'promotion' => true],
+            ['skill' => 'S009', 'promotion' => true],
+            ['skill' => 'S010', 'promotion' => false],
+            ['skill' => 'S011', 'promotion' => false],
+            ['skill' => 'S012', 'promotion' => false],
+        ],
+        'C003' => [
+            ['skill' => 'S013', 'promotion' => true],
+            ['skill' => 'S014', 'promotion' => true],
+            ['skill' => 'S015', 'promotion' => true],
+            ['skill' => 'S016', 'promotion' => false],
+        ],
+        'C004' => [
+            ['skill' => 'S017', 'promotion' => true],
+            ['skill' => 'S018', 'promotion' => true],
+            ['skill' => 'S019', 'promotion' => true],
+            ['skill' => 'S020', 'promotion' => false],
+        ],
+        'C005' => [
+            ['skill' => 'S021', 'promotion' => true],
+            ['skill' => 'S022', 'promotion' => true],
+            ['skill' => 'S023', 'promotion' => true],
+            ['skill' => 'S024', 'promotion' => false],
+        ],
+    ];
+
+    public function __construct(EntityManagerInterface $entityManager)
+    {
+        $this->entityManager = $entityManager;
+    }
+
+    public function populate()
+    {
+        $manager = $this->entityManager;
+        $this->loadCategoriesSkills();
+        $manager->flush();
+    }
+
+    private function loadCategoriesSkills(): void
+    {
+        foreach (self::CATEGORIES_SKILLS_MAPPING as $category => $items) {
+            // Adds categories
+            $citizenProjectCategory = new CitizenProjectCategory(
+                self::categoryProvider()[$category], BaseEventCategory::ENABLED
+            );
+            $this->entityManager->persist($citizenProjectCategory);
+
+            // Adds skills
+            foreach ($items as $item) {
+                $citizenProjectSkill = new CitizenProjectSkill(self::skillProvider()[$item['skill']]);
+                $this->entityManager->persist($citizenProjectSkill);
+
+                // Adds the categories / skills relation
+                $citizenProjectCategorySkill = new CitizenProjectCategorySkill(
+                    $citizenProjectCategory, $citizenProjectSkill, $item['promotion']
+                );
+
+                $this->entityManager->persist($citizenProjectCategorySkill);
+            }
+        }
+    }
+
+    private static function categoryProvider(): array
```

Comment:
Can't we reuse some fixtures constants like https://github.com/EnMarche/en-marche.fr/blob/master/src/DataFixtures/ORM/LoadCitizenInitiativeCategoryData.php#L11?

Reply:
We can, but I wanted to separate the fixtures used only in local/dev and the migrations data for production.

Reply:
The thing is that it adds all that in the cache anyway, so we could use prod values for dev fixtures, but this change would break some tests. It might not worth it, but if there aren't so many broken tests I suggest to do so.

Reply:
I think it's a good idea for another features, in this case, this service will be removed as soon as the migration is run in production, and I think also this change would break somes tests.